vshard/storage/init.lua

```diff
@@ -1808,6 +1809,9 @@ local function bucket_test_gc(bids)
     local _bucket = box.space._bucket
     local ref, bucket, status
     for i, bid in ipairs(bids) do
+        if M.rebalancer_transfering_buckets[bid] then
+            goto not_ok_bid
```

Comment:
It shouldn't happen normally, but could happen accidentally. For instance the config is wrong, or the user turned on auto-master and somehow got 2 nodes writeable. It might also happen that a master was demoted to a replica state while had a bucket send in progress. Then it will be a replica, but will have `M.rebalancer_transfering_buckets` set.

Reply:
Ok, I don't mind leaving it for safety